projects/common-platform-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/Handler.kt

```diff
@@ -44,7 +44,9 @@ class Handler(
         val courtCode = notification.message.hearing.courtCentre.code
 
         defendants.forEach { defendant ->
-            val matchRequest = defendant.toProbationMatchRequest()
+
+            val matchRequest = defendant.toProbationMatchRequest() ?: return
```

Comment:
This will return out of the handler altogether - would we rather continue looping in case there are other valid defendants?

You can use [return labels](https://kotlinlang.org/docs/returns.html#return-to-labels) in Kotlin to return to the forEach loop e.g.
```suggestion
            val matchRequest = defendant.toProbationMatchRequest() ?: return@forEach
```

Should probably do the same for the one further down too.